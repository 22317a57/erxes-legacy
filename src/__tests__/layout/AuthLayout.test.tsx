import { shallow } from 'enzyme';
import React from 'react';

import AuthLayout from '../../modules/layout/components/AuthLayout';

describe('AuthLayout component', () => {
  const defaultProps = {
    content: false
  };

  test('renders AuthLayout successfully', () => {
    shallow(<AuthLayout {...defaultProps} />);
  });
});
