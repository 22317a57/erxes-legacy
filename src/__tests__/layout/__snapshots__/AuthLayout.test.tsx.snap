// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthLayout component snapshot matches 1`] = `
<div
  className="sc-jnlKLf dxEFgG"
>
  <div
    className="sc-fYxtnH ibjvcA"
  >
    <div
      className="container"
    >
      <div
        className="col-md-6"
      >
        <div
          className="sc-tilXH dxHcPp"
        >
          <img
            alt="erxes"
            src="/images/logo.png"
          />
          <h1>
            Open Source Growth Marketing Platform
          </h1>
          <p>
            Marketing, sales, and customer service platform designed to help your business attract more engaged customers. Replace Hubspot with the mission and community-driven ecosystem.
          </p>
          <a
            href="http://erxes.io/"
          >
            Â« Go to home page
          </a>
        </div>
      </div>
      <div
        className="col-md-5 col-md-offset-1"
      />
    </div>
  </div>
</div>
`;
